glossary.md

```diff
@@ -0,0 +1,16 @@
+You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
```

Comment:
Nit: shouldn't this go at the bottom of the file, as it's meant to be an addition to this file based in the text?